### DISCUSSION TITLE
Option to restart container

### DISCUSSION THREAD

**Discussion by johnvpr** [blakeblackshear/frigate#10802] [Ideas]
Would it be nice to have an option to restart the container, as example the one they have on homebridge:
<img width="484" alt="image" src="https://github.com/blakeblackshear/frigate/assets/5040419/31788c2f-8c5a-4a61-af4c-1c4d4f4110bc">

https://github.com/homebridge/homebridge

**Comment by NickM-27** [comment]
What do you mean? There is a restart button in the ui

**Reply by johnvpr** [reply]
Yes, when you click click menu -> restart frigate, it will be nice to have a menu with multiple way of restart frigate,

**Reply by NickM-27** [reply]
Why? Frigate always restarts the entire container and that is what it expects to happen when it is starting up as well

**Reply by johnvpr** [reply]
Ok, my bad, sounds good!
